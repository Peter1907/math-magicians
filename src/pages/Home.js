import React from "react";

const Home = () => {
  const style = {
    paddingBottom: "10px",
    color: "#c6e6ff",
    fontSize: "20px",
  }
  return (
    <section className="home">
      <h2 style={{fontSize: "32px"}}>Welcome to our page!</h2>
      <p style={style}>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Enim necessitatibus pariatur quidem a eveniet, expedita, provident hic reprehenderit eum iste suscipit. Fuga odio quia fugit iste dicta quasi minus nostrum quis soluta illum expedita eligendi, maiores veniam mollitia alias consequuntur cumque nulla accusamus vitae exercitationem aut quam iure dolor aliquid. Placeat repellat quis sequi ab fugiat quaerat iste vel delectus laudantium illo consectetur illum quidem quibusdam inventore perspiciatis ea voluptatem, cum autem reiciendis sapiente nulla voluptas. Vel, ad voluptas ex dolor totam dolorem similique sed eveniet soluta aspernatur placeat nam officiis aliquam. Officia maiores sapiente distinctio cumque, eveniet eos saepe.</p>
      <p style={style}>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Placeat perferendis necessitatibus praesentium pariatur similique. Pariatur consectetur eveniet sunt, id, sit harum inventore quo in ullam magni ab ad perferendis ratione reiciendis modi voluptas numquam suscipit enim ipsam accusamus delectus excepturi tenetur. Nam totam fugit impedit voluptas laborum, et blanditiis quaerat officiis adipisci commodi! Vitae praesentium eveniet in ipsum natus dolores aut, quia, tenetur est similique ut excepturi neque atque itaque voluptatum alias, veritatis fugiat! Magni recusandae exercitationem nihil fugiat omnis eligendi saepe voluptatum, nesciunt reiciendis quia ratione tempora voluptatem consequatur itaque aliquam deserunt maiores? Vero sint numquam fugit minus nesciunt.</p>
    </section>
  )
};

export default Home
