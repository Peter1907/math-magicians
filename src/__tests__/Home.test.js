/* eslint-disable*/ 
import renderer from 'react-test-renderer';
import Home from '../pages/Home';


describe('Test components', () => {
it('Home test', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  })
});