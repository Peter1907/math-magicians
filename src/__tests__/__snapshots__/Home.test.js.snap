// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test components Home test 1`] = `
<section
  className="home"
>
  <h2
    style={
      Object {
        "fontSize": "32px",
      }
    }
  >
    Welcome to our page!
  </h2>
  <p
    style={
      Object {
        "color": "#c6e6ff",
        "fontSize": "20px",
        "paddingBottom": "10px",
      }
    }
  >
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ratione, et accusantium beatae reiciendis eveniet in, quidem aliquid dolorum nisi quis sit minima aspernatur? Accusantium quia a fugiat voluptatibus pariatur eveniet, doloremque illo ex esse hic optio numquam repellendus ab suscipit sint eius facere reprehenderit tenetur minima cumque nam. Saepe odit velit quasi culpa unde delectus molestiae exercitationem qui, fugit, fugiat repellat deserunt recusandae minus libero rem necessitatibus animi, voluptates eum. Magni expedita soluta quam vel qui obcaecati facere non, harum vero ratione cum autem mollitia veniam pariatur cumque velit veritatis corrupti sed voluptates maiores esse magnam animi quae? Voluptatem, quibusdam?
  </p>
  <p
    style={
      Object {
        "color": "#c6e6ff",
        "fontSize": "20px",
        "paddingBottom": "10px",
      }
    }
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa, magni hic eius, tempora iste ipsam mollitia non aspernatur inventore necessitatibus tempore nemo provident modi. Deleniti aut officiis quisquam veritatis, illo optio animi ullam inventore nesciunt debitis consequatur illum magnam minus a neque, velit tempore, earum unde dolore natus. Architecto cumque et enim porro animi repudiandae error quam illum culpa distinctio mollitia, suscipit cum eos eius non excepturi saepe dolores, delectus ipsum totam? Error harum eum, voluptate a cum velit cupiditate incidunt, iure beatae facilis omnis, sunt vero! Fuga, at vitae ducimus architecto, cum sint hic sunt voluptates inventore doloribus molestias.
  </p>
</section>
`;
